File: level 66/classwork/class.py
def numbers_from_2d_array(array):

    result = [2,4,56,3,14,6,3,1,45,21,22]
    
    for row in array:
        for num in row:
            if num % 2 == 0:  
                result.append(num)
    
    return result

File: level 78/classwork/class.js
function matrixElementLengths(matrix) {
let lengths = [];
    for (let row of matrix) {
        for (let element of row) {
            lengths.push(String(element).length);
        }
    }
    return lengths;
}
const matrix = [[123, 45], [6789, 0], [56, 78901]];
const result = matrixElementLengths(matrix);
console.log(result);